org.mixedrealitytoolkit.accessibility/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "org.mixedrealitytoolkit.accessibility",
-  "version": "1.0.0-development",
+  "version": "1.0.1-development",
   "description": "*** Please note ***\nThe MRTK3 accessibility features are in early preview and may not contain all planned features. Early preview releases also may undergo significant, breaking architectural changes before release.\n\nFeatures and subsystem to enable accessibility in mixed reality experiences.",
   "displayName": "MRTK Accessibility Early Preview",
   "msftFeatureCategory": "MRTK3",
-  "author": "Microsoft",
+  "author": "Mixed Reality Toolkit Contributors",
```

Comment:
While we're at it, do we want to remove the "Microsoft®" from the product names in our AssemblyInfo files?

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/44440e0291e48bd7cb53483e7fb2fcc7e015a166/org.mixedrealitytoolkit.accessibility/AssemblyInfo.cs#L9

Reply:
Aggreed!  @shaynie  please make this change part of this PR